packages/vscode-extension/package.json

```diff
@@ -760,7 +760,7 @@
         "background": {
           "activeOnStart": true,
           "beginsPattern": "[nodemon] starting",
-          "endsPattern": "listening|[nodemon] app crashed"
+          "endsPattern": "restify listening to|Bot/ME service listening at|[nodemon] app crashed"
```

Comment:
To keep backward compatibility, no need to change this.
Modifying in tasks.json is enough since $teamsfx-bot-watch is not used for new project.

Reply:
Reverted this change.